src/bins/rmb-relay.rs

```diff
@@ -131,7 +138,7 @@ async fn app(args: &Args) -> Result<()> {
     // we use 6 hours cache for twin information because twin id will not change anyway
     // and we only need twin public key for validation only.
     let twins = SubstrateTwinDB::<RedisCache>::new(
-        &args.substrate,
+        args.substrate.clone(),
```

Comment:
because substrate is behind a shared reference, for some reason args is a reference.

Reply:
Yeah, because it seems that I call app(&args) instead of app(args) which can be easily changed in rmb-relay since i can already see in the rmb-peer than we pass the object itself, not a ref